File: Javascript/command.js
//1.조건문 
// if(조건식) 참일때 실행문; 
// if(조건식) {참일때 실행문} else {거짓일때 실행문};
let month = prompt("월 입력 :", "");

if(month==1||month==3||month==5||month==7||month==8||month==10||month==12){
    console.log(`${month}월은 31일까지 있습니다.`);
}else if(month==2){
    console.log(`${month}월은 28일까지 있습니다.`);
}else if(month==4 || month==6 || month==9 || month ==11){
    console.log(`${month}월은 30일까지 있습니다.`);
}else{
    console.log(`${month}월이 맞는지 확인하세요`);
}
//swich(변수명) {
//     case 변수값: 실행문
//                  break;
//     case 변수값: 실행문
//                  break;
//     default: 기타실행문;
// }
switch(month){
    case '1':
    case '3':
    case '5':
    case '7':
    case '8':
    case '10':
    case '12': console.log(`${month}월은 31일까지 있습니다.`);
             break;
    case '2': console.log(`${month}월은 28일까지 있습니다.`);
             break;
    case '4':
    case '6':
    case '11':console.log(`${month}월은 30일까지 있습니다.`);
             break;
    default : console.log(`${month}월이 맞는지 확인하세요`);
}

//반복문
//초기값;                         초기값;            for(초기값; 조건식; 증감식){
//while(조건식){                  do{                       실행문;
//    참일때 실행문;                실행문;           }
//    증감식;                       증감식;          for( 데이터 in|of 리스트데이터){
//}                                }while(조건식);         실행문; }

let index=1;
let sum=0;
while(index < 11){
    sum += index; //sum = sum+index;
    index++;  //index=index+1;
}
console.log(`1~10 합 : ${sum}`);

index=1;
sum=0;
do{
   sum += index;
   index++;
}while(index < 11);
console.log(`1~10 합 : ${sum}`);

sum=0;
for(index=1; index < 11; index++){
    sum+=index;
}
console.log(`1~10 합 : ${sum}`);

//기타 : return - 함수종료(반환), break - 반복문 종료, continue-반복문으로 돌아가기
sum=0;
for(index=1; index < 11; index++){
    if( index%2 ==0) continue;
    sum+=index;
}
console.log(`1~10 홀수 합 : ${sum}`);
//nexted for문 이용 구구단 출력 해보기~~~

File: Python/command.py
# if 조건식 :
#     참일때 실행문
# elif 조건식2 :
#     조건식2가 참일때 실행문
# else :
#     조건식, 조건식2 거짓일때 실행문
# month = input("월 입력 : ")
# if month.isdecimal() :
#     m = int(month)
#     if m==1 or m==3 or m==5 or m==7 or m==8 or m==10 or m==12 :
#         print("{0}월은 31일까지 있습니다.".format(m))
#     elif m==2 :
#         print("{0}월은 28일까지 있습니다.".format(m))
#     elif m==4 or m==6 or m==9 or m==11 :
#         print("{0}월은 30일까지 있습니다.".format(m))
#     else :
#         month=input("월은 1~12사이 값을 입력 : ")    
# else:
#      month=input("월은 1~12사이 값을 입력 : ")


#list type : [str,int, float, boolean, list] 
#           +, *(iteration), len() 
# append(), insert(), del, pop(), remove(), clear()
list_a = [1,2,1,3]
list_b=[4,5,6]
print(list_a+list_b,"+연산후 list_a: ", list_a)
print(list_a.append(4), "append후 list_a ", list_a) #마지막 index+1에 추가
print(list_a.insert(1,5), "insert후 list_a ", list_a) #해당 index에 삽입
print(list_a.pop(1), "pop(1) 삭제 후  list_a", list_a) #해당 index  삭제
print(list_a.remove(1),"remove(1) 1인 값 삭제 후 list_a", list_a) #첫번째 만나는 값 삭제

#for 반복자 in 반복할 수 있는 데이터(list, dictionary, string, range()) :
#   실행문
index=0
for data in list_a:
    print("list_a[{0}] : {1}".format(index, data))
    index +=1




